transfer.py

```diff
@@ -52,7 +52,7 @@ def copy_object(videofront_key, marsha_bucket, marsha_key):
     marsha_s3.copy(copy_source, marsha_bucket, marsha_key)
 
 
-def get_or_create_video(videofront_key, course_key, xblock_id, uuid):
+def get_or_create_video(instance, videofront_key, course_key, xblock_id, uuid):
```

Comment:
Are you sure you will always have this `instance` parameter` ? I don;t see it from where this function is called.

Reply:
It comes from the csv file that you import. It will fail if the csv file does not have an `instance` column but that's what we want.